gapic/src/main/com/google/gapid/util/Colors.java

```diff
@@ -42,6 +44,18 @@ public static RGB rgb(double r, double g, double b) {
     return new RGB((int)(r * 255), (int)(g * 255), (int)(b * 255));
   }
 
+  public static RGB getRandomColor(int index) {
+    float hue = (float)Math.toDegrees(index * GOLDEN_ANGLE) % 360;
+    return new RGB(hue, 1f, 1f);
```

Comment:
www.shadertoy.com
```glsl
const float GOLDEN_ANGLE = 2.39996322972865332f;
const float COLOR_INTENSITY = 0.2;
const float PI = 3.141592654;
vec3 hsv2rgb(vec3 c) {
    vec4 K = vec4(1.0, 2.0 / 3.0, 1.0 / 3.0, 3.0);
    vec3 p = abs(fract(c.xxx + K.xyz) * 6.0 - K.www);
    return c.z * mix(K.xxx, clamp(p - K.xxx, 0.0, 1.0), c.y);
}
void mainImage(out vec4 fragColor, in vec2 fragCoord) {
	ivec2 cell = ivec2(fragCoord.xy / vec2(64, 64));
	vec2 uv = vec2(cell);
	float angle = uv.x * GOLDEN_ANGLE;
	vec3 rgb;
	if (fragCoord.y > (iResolution.y/2.0)) {
		rgb = sin(angle + PI*vec3(0.0, 1.0, 2.0)/3.0);
		rgb += 1.0;
		rgb = normalize(rgb);
	} else {
		float hue = angle / (2.0*PI);
		vec3 hsv = vec3(hue, 1.0, 0.8);
		rgb = hsv2rgb(hsv);
	}
	fragColor = vec4(mix(vec3(1.0), rgb, COLOR_INTENSITY), 1.0);
}
```

Top: original, Bottom: new.

Personally, I feel that the top has a more neutral, pastel palette.